OpenSim/Common/APDMDataReader.cpp

```diff
@@ -67,21 +67,25 @@ APDMDataReader::extendRead(const std::string& fileName) const {
     bool newFormat = false;
     if (tokens[0] == "Format=7") {
         newFormat = true;
-        std::vector<std::string> labels;
+        dataRate = 128; // Will fix on reading first 2 rows
```

Comment:
Why is this initialized to 128 instead of NaN or some other way to detect an error?

Reply:
@aseth1 the old format had an explicit datarate line, new format has none but provide timestamp as a longlong integer. We'll need to compute data rate from actual timestamps of the first two rows of data, so this part is WIP.